code/modules/xenomorph/xeno_structures.dm

```diff
@@ -67,6 +67,10 @@
 		UnregisterSignal(associated_hive, list(COMSIG_HIVE_XENO_MOTHER_PRE_CHECK, COMSIG_HIVE_XENO_MOTHER_CHECK))
 		associated_hive.xeno_message("<span class='xenoannounce'>A resin silo has been destroyed at [silo_area]!</span>", 2, TRUE)
 		associated_hive = null
+	for(var/i in contents)
+		var/atom/movable/AM = i
```

Comment:
Thinking on this, we know that only humans can make a silo.

Thus I think we should only return bodies that have a chance of being revived. 

So I think instead of `AM` it needs to be a `human` and to `check_tod()` on each of them. `QDEL()` the ones that can't revive.

Reply:
Why would you delete unrevivables theres no reason for that

Reply:
The assumption being that since they are fully dead, their body has broken down into sludge and doesn't truly exist anymore.

Reply:
It probably wouldn't be worth the effort to add a check for if they can be revived, and then having their items be stripped, which can then be expelled on silo death.

Reply:
New suggestion. make them husked like. to show how much they've dissolved in it.

Reply:
```
for(var/thing in contents)
		if(!ishuman(thing))
			continue
		var/mob/living/carbon/human/dead_guy = thing
		dead_guy.icon_state = "husk_s"
		dead_guy.update_icon()
		dead_guy.forceMove(loc)
```